src/ape_ethereum/provider.py

```diff
@@ -1326,9 +1336,7 @@ class EthereumNodeProvider(Web3Provider, ABC):
     name: str = "node"
 
     # NOTE: Appends user-agent to base User-Agent string.
-    request_header: dict = {
-        "User-Agent": construct_user_agent(str(HTTPProvider)),
-    }
+    request_header: dict = {"User-Agent": f"EthereumNodeProvider/web3.py/{web3_version}"}
```

Comment:
note to reviewer: this basically makes it the same as it was before, v7 changes this method to make it more web3-specific by including a required module arg